src/lib/one-graph/client.js

```diff
@@ -0,0 +1,835 @@
+          sessionId
+        }
+      }
+    }
+  }
+  
+query CLISessionEventsQuery(
+  $nfToken: String!
+  $sessionId: String!
+  $first: Int!
+) {
+  oneGraph(
+    auths: { netlifyAuth: { oauthToken: $nfToken } }
+  ) {
+    netlifyCliEvents(sessionId: $sessionId, first: $first) {
+      __typename
+      createdAt
+      id
+      sessionId
+      ... on OneGraphNetlifyCliSessionLogEvent {
+        id
+        message
+        sessionId
+        createdAt
+      }
+      ... on OneGraphNetlifyCliSessionTestEvent {
+        id
+        createdAt
+        payload
+        sessionId
+      }
+    }
+  }
+}
+  
+mutation AckCLISessionEventMutation(
+  $nfToken: String!
+  $sessionId: String!
+  $eventIds: [String!]!
+) {
+  oneGraph(
+    auths: { netlifyAuth: { oauthToken: $nfToken } }
+  ) {
+    ackNetlifyCliEvents(
+      input: { eventIds: $eventIds, sessionId: $sessionId }
+    ) {
+      events {
+        id
+      }
+    }
+  }
+}
+
+query AppSchemaQuery(
+  $nfToken: String!
+  $appId: String!
+) {
+  oneGraph(
+    auths: { netlifyAuth: { oauthToken: $nfToken } }
+  ) {
+    app(id: $appId) {
+      graphQLSchema {
+        appId
+        createdAt
+        id
+        services {
+          friendlyServiceName
+          logoUrl
+          service
+          slug
+          supportsCustomRedirectUri
+          supportsCustomServiceAuth
+          supportsOauthLogin
+        }
+        updatedAt
+      }
+    }
+  }
+}
+
+mutation UpsertAppForSiteMutation(
+  $nfToken: String!
+  $siteId: String!
+) {
+  oneGraph(
+    auths: { netlifyAuth: { oauthToken: $nfToken } }
+  ) {
+    upsertAppForNetlifySite(
+      input: { netlifySiteId: $siteId }
+    ) {
+      org {
+        id
+        name
+      }
+      app {
+        id
+        name
+        corsOrigins
+        customCorsOrigins {
+          friendlyServiceName
+          displayName
+          encodedValue
+        }
+      }
+    }
+  }
+}
+
+mutation CreateNewSchemaMutation(
+  $nfToken: String!
+  $input: OneGraphCreateGraphQLSchemaInput!
+) {
+  oneGraph(
+    auths: { netlifyAuth: { oauthToken: $nfToken } }
+  ) {
+    createGraphQLSchema(input: $input) {
+      app {
+        graphQLSchema {
+          id
+        }
+      }
+      graphqlSchema {
+        id
+        services {
+          friendlyServiceName
+          logoUrl
+          service
+          slug
+          supportsCustomRedirectUri
+          supportsCustomServiceAuth
+          supportsOauthLogin
+        }
+      }
+    }
+  }
+}`
+
+const fetchPersistedQuery = async (authToken, appId, docId) => {
+  const response = await fetchOneGraph(authToken, ONEDASH_APP_ID, internalOperationsDoc, 'PersistedQueryQuery', {
+    nfToken: authToken,
+    appId,
+    id: docId,
+  })
+
+  const persistedQuery = response.data && response.data.oneGraph && response.data.oneGraph.persistedQuery
+
+  return persistedQuery
+}
+
+const fetchCliSessionEvents = async ({ appId, authToken, sessionId }) => {
+  const first = 1000
```

Comment:
can you document the magic number here? why 1000 -> is this a pagination or what does this first do? 

Maybe you could rename to a more descriptive variable name